poisson_lib.py

```
@@ -74,10 +74,10 @@ def run_poisson_regression(
 
 
 def run_negative_binomial_regression(
-    y: pd.Series | np.ndarray,
+    y: Union[pd.Series, np.ndarray],
```

Ruff (0.14.10)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)